README_EN.md

```diff
@@ -0,0 +1,233 @@
+# Rexxar iOS
+
+[![Test Status](https://travis-ci.org/douban/rexxar-ios.svg?branch=master)](https://travis-ci.org/douban/rexxar-ios)
+[![IDE](https://img.shields.io/badge/XCode-8-blue.svg)]()
+[![iOS](https://img.shields.io/badge/iOS-7.0-green.svg)]()
+[![Language](https://img.shields.io/badge/language-ObjC-blue.svg)](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html)
+
+**Rexxar** is a hybrid library for mobile application development. Now it supports iOS and Android platform. `Rexxar iOS` is Rexxar's Container in iOS.
+
+With Rexxar, you can develop mobile application by traditional web technologies including javascript, html and css. Rexxar Container has not any requirements of web front side. Our demo in the web front side is implemented by [React](https://facebook.github.io/react/). You can use your own web front side framework to develop the application in Rexxar Container.
+
+Rexxar iOS supports iOS 7.0 and above systems.
+
+
+## Rexxar
+
+About Rexxar's integral introduction, please check this article: [豆瓣的混合开发框架 -- Rexxar](http://lincode.github.io/Rexxar-OpenSource). Sorry about these is only Chinese version at present.
```

Comment:
No need to add this line if you are sure your audience don't understand Chinese.